app/views/bus_stops/_form.haml

```diff
@@ -14,30 +14,30 @@
       %tr
         %td= f.label :hastus_id, 'Hastus ID'
         %td= stop.hastus_id
-      - if current_user.admin?
         %tr
           %td= f.label :routes
           %td= stop.route_list
-      %tr
-        %td= f.label :completed, 'Completed?'
-        %td= f.check_box :completed
       %tr
         %td= f.label :garage_responsible
         %td= f.select :garage_responsible, %w[SATCo UMTS VATCo], include_blank: true
       %tr
         %td= f.label :state_road
-        %td= f.check_box :state_road
+        %td= f.select :state_road, %w[Yes No], include_blank: true
     - BusStop::SUPER_HASH.each do |category, hash|
       %th
         = f.label category
         - hash.each do |name, options|
           %tr
             %td= f.label name
-            %td
-              - if options.kind_of?(Array)
-                = f.select name, options, include_blank: true
-              - else
-                = f.check_box name
+            - if options.kind_of?(Array)
+              %td= f.select name, options, include_blank: true
+            - else
+              %td= f.select name, %w[Yes No], include_blank: true
```

Comment:
Have we confirmed that this saves correctly? Does `"No"` coming over the wire translate to a false boolean value being saved?

Reply:
Good call, we didn't even change the field types!

Reply:
However, I have confirmed that `%td= f.select name, %w[Yes No], include_blank: true` saves correctly as a string for both yes and no.

Reply:
Or were we hoping the inverse was true? As in we wanted `"No"` to translate to `false`, because that did not work.